src/azimuth/gui/screen.c

```diff
@@ -52,6 +51,19 @@ static int current_screen_height = AZ_SCREEN_HEIGHT;
 static float current_screen_scale = 1.0f;
 static float current_screen_xoffset = 0;
 static float current_screen_yoffset = 0;
+static double NANOSECONDS_PER_COUNT = 1000000000;
```

Comment:
The usual style in this codebase is to only use `ALL_CAPS_NAMES` for constants.  For consistency, could we rename this to `nanoseconds_per_count`?